src/storage/src/hummock/compactor/iterator.rs

```diff
@@ -30,19 +31,22 @@ use crate::hummock::compactor::task_progress::TaskProgress;
 use crate::hummock::iterator::{Forward, HummockIterator};
 use crate::hummock::sstable_store::{BlockStream, SstableStoreRef};
 use crate::hummock::value::HummockValue;
-use crate::hummock::{Block, BlockHolder, BlockIterator, HummockResult};
+use crate::hummock::{BlockHolder, BlockIterator, HummockResult, TableHolder};
 use crate::monitor::StoreLocalStatistic;
 
+const MAX_RETRY_TIME: u64 = 600; // 10min
```

Comment:
I think we define more and more constants , it is best to manage them uniformly (another PR)

Reply:
ok